wazimap_ng/profile/serializers/metrics_serializer.py

```diff
@@ -28,11 +28,20 @@ def sibling(profile_key_metric, geography):
     return None
 
 def absolute_value(profile_key_metric, geography):
-    indicator_data = IndicatorData.objects.filter(indicator__profilekeymetrics=profile_key_metric, geography=geography)
+    indicator_data = IndicatorData.objects.filter(
+        indicator__profilekeymetrics=profile_key_metric, geography=geography
+    )
     if indicator_data.count() > 0:
         subindicator = get_subindicator(profile_key_metric)
-        data = indicator_data.first().data # TODO what to do with multiple results
-        return data["subindicators"][subindicator]
+        data = indicator_data.first().data  # TODO what to do with multiple results
+        subindicators_data = data.get("subindicators")
+        if subindicators_data:
+            absolute_value = subindicators_data.get(subindicator)
```

Comment:
You can and should use a default here for `.get` see the docs: https://docs.python.org/3.9/library/stdtypes.html#dict.get

Reply:
![absolute](https://user-images.githubusercontent.com/29105699/111844484-f0665b80-8924-11eb-8bc0-2d64c0c60d8c.png)

correct me if i am wrong, you want this to have default as **'N/A'** and if so then what will be the expected then this

**_if_** below should have extra **_and_** operator for checking instance as dict too? because 'N/A' is string
```
  if subindicators_data:
            absolute_value = subindicators_data.get(subindicator)
```